internal/librarian/createreleaseartifacts.go

```diff
@@ -153,13 +153,13 @@ func copyMetadataFiles(outputRoot string, releases []LibraryRelease, languageRep
 	if err := languageRepo.Checkout(finalRelease.CommitHash); err != nil {
 		return err
 	}
-	sourceStateFile := filepath.Join(languageRepo.Dir, config.GeneratorInputDir, pipelineStateFile)
+	sourceStateFile := filepath.Join(languageRepo.Dir, config.LibrarianDir, pipelineStateFile)
```

Comment:
I made this change because the pipeline state file should live in `.librarian` directory.